src/commonMethods/webKitPerformanceCommonFunctions.js

```diff
@@ -0,0 +1,61 @@
+let samples = []
+
+/**
+ * This function resets the Samples array
+ */
+export const initializeSamplesArray = function() {
+  samples = []
+}
+
+/**
+ * This function fetches the FPS value
+ * @returns {Samples Array}
+ */
+export const fetchFPS = function() {
+  this.$thunder.api.WebKitBrowser.fps()
+    .then(result => {
+      if (isNaN(result) === false) {
+        samples.push(result)
+        return true
+      } else {
+        this.$log('Returned FPS value from Framework is not a number')
+        throw new Error('Returned FPS value from Framework is not a number')
+      }
+    })
+    .catch(err => err)
+  return samples
+}
+
+/**
+ * This function calculates the average of FPS samples collected in fetchFPS function
+ * @returns {results}
+ */
+export const calcAvgFPS = function() {
```

Comment:
this is a good example of a generic function that we could incorporate into Storm core (as `calculateAverage` probably, since it's not limited to calculating avg FPS)